modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,62 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
 == Required account privileges
 
-Installing a cluster to Nutanix requires an account with administrative privileges to read and create the required resources.
+The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster, as well as to maintain the daily operation of it. The following options are available to you:
+
+* You can use a local Prism Central user account with administrative privileges. Using a local account is the simplest way to grant access to an account with the required permissions.
+* If your organization’s security policies require that you use a more restrictive set of permissions, use the permissions in the following table to create a custom Cloud Native role in Prism Central. You can then assign the role to a user account that is a member of a Prism Central authentication directory. When assigning entities to the role, ensure that the user can only access the Prism Element and subnet that are required to deploy the virtual machines.
++
+For more information, see the Nutanix documentation about creating a link:https://opendocs.nutanix.com/guides/cloud_native_role/[Custom Cloud Native role] and link:https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Security-Guide:ssp-ssp-role-assignment-pc-t.html[assigning a role].
```

Comment:
@mburke5678 . What do you think about linking to external (but partner) docs?

Reply:
@dfitzmau We do use external docs. Not sure what the official rule is, if there is one. However, it leads to (possibly) outdated links, like this one: [Nutanix AOS Security Guide](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Security-Guide-v6_1:mul-security-ssl-certificate-pc-t.html)
@mjpytlak

Reply:
@dfitzmau We are able to point to external provider docs. @mburke5678 I am not sure why the link resolves to an outdated version, as there is no version in the link that Nutanix provided me. It would appear that this a function of the Nutanix doc set. While not ideal, the Nutanix doc set does let users know about the outdated version and provides a link to latest.

Reply:
:heavy_check_mark: . NP. Please consider the maintenance overhead that might exist when a link gets outdated and so on.